### PR TITLE
 TorchDynamo: Add convolution binary fusion for cpu in inference mode

### PR DESCRIPTION
**Pull request by XiaobingSuper**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #88414
* #88412
* #88403
* #88048
* #87435
* #87066
* #87065
* __->__ #87064

cc @mlazos @soumith @voznesenskym @yanboliang @penguinwu @anijain2305 @EikanWang @jgong5 @Guobing-Chen @chunyuan-w @zhuhaozhe @blzheng @Xia-Weiwen @wenzhe-nrv @jiayisunx @peterbell10 @jansel @lezcano @fdrocha

### CODE REVIEW

File: torch/_inductor/overrides.py [hunk 1]
```diff
@@ -133,6 +191,41 @@ def fuse_conv_unary_eval(conv: nn.Module, unary: nn.Module):
...
+
+
+def check_node_is_binary(node):
+    if (
+        (node.op == "call_function" and node.target in [torch.add, torch.sub])
+        or (node.op == "call_function" and node.target in [operator.add, operator.sub])
+        or (
+            node.op == "call_method"
+            and node.target in [torch.Tensor.add, torch.Tensor.sub]
+        )
```
Comment (jgong5) [inline torch/_inductor/overrides.py:223]: Simpler to just return the value of this boolean expression instead of an extra "if-else" here?

**Review by jgong5 — CHANGES_REQUESTED** [review]

**Review by jgong5 — APPROVED** [review]

**Comment by XiaobingSuper** [comment]
@jansel , could you help review this PR? Thanks!

**Comment by XiaobingSuper** [comment]
@jansel

**Review by jansel — APPROVED** [review]

**Pull request closed without merging**